### ISSUE TITLE
Visualize existing resources in OCI

### ISSUE THREAD

**Issue by kirankumar-badiger** [oracle/oci-designer-toolkit#599]
Is there any documentation on how we can use this visualize oci infrastructure resources

**Comment by toxophilist** [comment]
@kirankumar-badiger do you mean Query OCI

**Comment by kirankumar-badiger** [comment]
@toxophilist yes

**Comment by toxophilist** [comment]
@kirankumar-badiger Just use the File->Query OCI menu option.

**Comment by kirankumar-badiger** [comment]
Yes i am able to visualize the infrastructure, we are working on internal tool for OCI to visualize the deployed infrastructure

**Comment by toxophilist** [comment]
@kirankumar-badiger Oracle internal

**Comment by toxophilist** [comment]
@kirankumar-badiger Are you questions answered ?

**Comment by kirankumar-badiger** [comment]
@toxophilist Is there any documentation to use only **query oci** feature.
